### PR TITLE
Update User.js

### PR DESCRIPTION
**Pull request by Hetchy**

### CODE REVIEW

File: backend/models/User.js [hunk 1]
```diff
@@ -2,32 +2,28 @@ import mongoose from 'mongoose';
...
-            type: String,
-            required: true
-        },
-        description: {
-            type: String,
-            required: true
-        },
+        email: {type: String, required: true},
+        username: {type: String, required: true, unique: true, trim: true},
+        password: {type: String, required: true},
```
Comment (Peedee2002) [inline backend/models/User.js:7]: we dont need this?

File: backend/models/User.js [hunk 2]
```diff
@@ -2,32 +2,28 @@ import mongoose from 'mongoose';
...
-            unique: true
-        },
-        password: {
-            type: String,
-            required: true
-        },
-        description: {
-            type: String,
-            required: true
-        },
```
Comment (Peedee2002) [inline backend/models/User.js:29]: when you get to sign in with google, you will notice that you will need to store something called a `sub` which is the token for that user. That may be their "password" in a sense, but i dont particularly like that.

File: backend/models/User.js [hunk 3]
```diff
@@ -2,32 +2,28 @@ import mongoose from 'mongoose';
...
+        email: {type: String, required: true},
+        username: {type: String, required: true, unique: true, trim: true},
+        password: {type: String, required: true},
+        firstName: {type: String, required: true, trim: true},
```
Comment (mi-mott) [inline backend/models/User.js]: I think we also need a way to store the token right?

**Review by mi-mott — CHANGES_REQUESTED** [review]
After logging in with google, need to be able to store that to identify the user

File: backend/models/User.js (hunk 3 above)
Comment (Hetchy) [reply backend/models/User.js]: yeah that's the "sub"

**Review by Peedee2002 — APPROVED** [review]

**Pull request merged**